mesh/nurbs.hpp

```diff
@@ -262,6 +267,19 @@ class NURBSPatch
    NURBSPatch(const KnotVector *kv0, const KnotVector *kv1,
               const KnotVector *kv2, int dim);
 
+   /** Create a bivariate NURBS patch with given control points. See n-variate overload for additional notes. */
```

Comment:
There are some long lines here, > 80 columns.

Reply:
I've enforced 80 columns for the documentation comments in the latest version.